OpenFGA.g4

```diff
@@ -1,20 +1,20 @@
 grammar OpenFGA;
 
-main: modelHeader typeDefs newline*;
+main: modelHeader typeDefs newline?;
 
 indentation: '  ' | '	';
 
-modelHeader: 'model' (newline+ multiLineComment)? newline indentation 'schema' spacing schemaVersion;
+modelHeader: (multiLineComment newline)? 'model' (newline+ multiLineComment)? newline indentation 'schema' spacing schemaVersion;
```

Comment:
Since newline is now defined as `'\n'+;`, do we still need to define `newline+` in `(newline+ multiLineComment)?` ?

Reply:
Fixed